payments/enums.py

```diff
@@ -70,12 +70,19 @@ class OrderStatus(TextChoices):
     ERROR = "error", _("Error")
     PAID = "paid", _("Paid")
     PAID_MANUALLY = "paid_man", _("Paid manually")  # max 9 chars
+    REFUNDED = "refunded", _("Refunded")
 
     @classmethod
     def get_paid_statuses(cls):
         return (cls.PAID, cls.PAID_MANUALLY)
 
 
+class OrderRefundStatus(TextChoices):
```

Comment:
Could add some documentation where and how the statuses change. Accepted/rejected is determined by Bambora, right? So make it clear that the issue is not asking an approval from the customer.

Reply:
In some cases, the [refund](https://payform.bambora.com/docs/web_payments/?page=full-api-reference#refund-payment-return) is sent as an email to the customer and they have to accept it:
> _email a confirmation email is sent to the customer and customer must accept it_

So there _could_ be the case when the user rejects it, but we don't get any additional details.

Reply:
ok! This process could be documented briefly either here or the status flow chart

Reply:
@haavikko you can find the flow on the [payments docs](https://github.com/City-of-Helsinki/berth-reservations/blob/688bab2b015578d922c68889432877c54f08ddfc/docs/payments.md)

<img width="1106" alt="image" src="https://user-images.githubusercontent.com/15201480/110921193-f48fe900-8326-11eb-8a47-545a5de4961e.png">